src/System.Collections/src/System.Collections.csproj

```diff
@@ -6,6 +6,9 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <Configurations>netcoreapp-Unix-Debug;netcoreapp-Unix-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release;uapaot-Windows_NT-Debug;uapaot-Windows_NT-Release</Configurations>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netcoreapp-Windows_NT-Debug|AnyCPU'">
```

Comment:
This can be achieved with `Condition="'$(TargetGroup)'  == 'netcoreapp' AND $(TargetsWindows)"` however why don't set LangVersion to latest always?

Reply:
A better question is why whoever changed the code in Stack and Queue to use a 7.1 language feature didn't checkin the project file that they had to have changed. 

As for why the condition is constructed that way, that's what visual studio chose to do. Changing it to latest is fine if that's ok with you. Does it even need to be conditional?

Reply:
I don't think it needs to be conditional, we can just set it for all configurations.